src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/VariantsToTable.java

```diff
@@ -0,0 +1,430 @@
+ * non-reference genotypes), NSAMPLES (number of samples), NCALLED (number of called samples),
+ * GQ (from the genotype field; works only for a file with a single sample), and MULTI-ALLELIC
+ * (is the record from a multi-allelic site).  </p>
+ *
+ * </p>
+ *
+ * <h3>Input</h3>
+ * <p>
+ * <ul>
+ *     <li>A VCF file</li>
+ *     <li>A list of -F fields to write</li>
+ * </ul>
+ * </p>
+ *
+ * <h3>Output</h3>
+ * <p>
+ * A tab-delimited file containing the values of the requested fields in the VCF file
+ * </p>
+ *
+ * <h3>Usage example</h3>
+ * <pre>
+ *     ./gatk-launch \
+ *     -R reference.fasta
+ *     -T VariantsToTable \
+ *     -V file.vcf \
+ *     -F CHROM -F POS -F ID -F QUAL -F AC \
+ *     -o results.table
+ * </pre>
+ * <p>would produce a file that looks like:</p>
+ * <pre>
+ *     CHROM    POS ID      QUAL    AC
+ *     1        10  .       50      1
+ *     1        20  rs10    99      10
+ *     et cetera...
+ * </pre>
+ *
+ * <h3>Caveat</h3>
+ * <p>If a VCF record is missing a value, then the tool by default throws an error, but the special value NA can
+ * be emitted instead if requested at the command line using --allowMissingData.</p>
+ *
+ * @author Mark DePristo
+ * @since 2010
+ */
+@CommandLineProgramProperties(
+        summary = "This tool is designed to extract fields from the VCF to a table format " +
+                "that is more convenient to work with in " +
+                "downstream analyses",
+        oneLineSummary = "Extract specific fields from a VCF file to a tab-delimited table",
+        programGroup = VariantProgramGroup.class
+)
+public final class VariantsToTable extends VariantWalker {
+
+    @Argument(fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME,
+            shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME,
+            doc="File to which results should be written")
+    private String out = null;
+
+    /**
+     * -F NAME can be any standard VCF column (CHROM, ID, QUAL) or any binding in the INFO field (e.g., AC=10).
+     * Note that to capture GENOTYPE (FORMAT) field values, see the GF argument.  This argument accepts any number
+     * of inputs.  So -F CHROM -F POS is allowed.
+     */
+    @Argument(fullName="fields", shortName="F", doc="The name of each field to capture for output in the table", optional=true)
+    private List<String> fieldsToTake = new ArrayList<>();
+
+    /**
+     * -GF NAME can be any binding in the FORMAT field (e.g., GQ, PL).
+     * Note this argument accepts any number of inputs.  So -GF GQ -GF PL is allowed.
+     */
+    @Argument(fullName="genotypeFields", shortName="GF", doc="The name of each genotype field to capture for output in the table", optional=true)
+    private List<String> genotypeFieldsToTake = new ArrayList<>();
+
+    /**
+     * By default this tool only emits values for fields where the FILTER field is either PASS or . (unfiltered).
```

Comment:
I think this should say "emits values for records where the FILTER field is" instead of "fields"